source/en/calibration_freshwater.rst

```diff
@@ -0,0 +1,150 @@
+
+Many different tools are available to create watersheds, and you can use whichever one you’re comfortable with. InVEST includes the tool :ref:`DelineateIt <delineateit>` as a simple, effective way of creating watersheds.
+
+Whichever tool you use, they generally require, at a minimum, a digital elevation model (DEM) raster, and a vector (like a shapefile or geopackage) containing the point location(s) to be used as outlets. In this case, the outlet will be the location of a gauge station that collected the observed data. The DEM must be the same one that is used as input to the InVEST freshwater model you’re calibrating.
+
+After running the delineation tool, look at the resulting watershed carefully to make sure that it appears correct. One common problem is that the delineated watershed is very tiny. This is usually caused by the outlet point not being located directly on a stream created by the delineation tool. To fix this, many delineation tools have a “snap” function, where you can specify a distance around the outlet point that the tool should look for a stream, and if one is found within that distance, the tool “snaps” the point to the stream, and delineates the watershed more accurately. If the tool does not have a snap feature, you can manually move the point to lie on the stream network generated by the delineation tool.
+
+Once the watershed is correctly generated, a GIS tool like Zonal Statistics is used to sum the relevant model result raster (such as sediment export or nutrient export) within the watershed. This summarized value is then compared with the observed data value. Alternatively, you can use the generated watershed as an input to the model, which will do the summarizing for you, creating a vector layer whose table contains the summarized values.
+
+Setting up the InVEST Python scripting environment
+==================================================
+
+1. Install Miniconda from: https://docs.anaconda.com/free/miniconda/
+2. Open Miniconda command line interface (CLI):
+	+ Create a new environment named after the InVEST version you plan to use it for (the example below is for InVEST 3.14.2, but yours is likely newer):
+		* ``conda create -y -c conda-forge -n invest3_14_2 python=3.11.0``
+	+ List all environments:
+		* ``conda info --envs``
+	+ Activate your newly created environment:
+		* ``conda activate invest3_14_2``
+	+ Install the InVEST Python package:
+		* ``conda install -c conda-forge natcap.invest``
+		* This command should install all dependencies because we’re using conda
+3. InVEST Python package version check
+	+ Launch python shell ``($ python)``:
+		* ``>>> import natcap.invest``
+		* ``>>> print(natcap.invest.__version__)``
+		* ``>>> exit()``
+	+ In this example, this installs InVEST v3.14.1, so update with pip:
+		* ``pip install natcap.invest==3.14.2``
```

Comment:
`conda install -c conda-forge natcap.invest` will always install the latest version, so there should not be a need to update right away. The suggestion to update the package using `pip` instead of `conda` kinda makes sense, but only if you know how those package managers differ and know that all your other dependencies don't need updating. I would probably just remove this `+` bullet point.